EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -81,6 +84,7 @@ void HcalRawToDigi::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<bool>("FilterDataQuality",true);
   desc.addUntracked<std::vector<int>>("FEDs", std::vector<int>());
   desc.addUntracked<bool>("UnpackZDC",true);
+  desc.addUntracked<bool>("UnpackZDCQIE10",true);
```

Comment:
this is a bogus switch: the HcalUnpacker.cc  implementation unpacks everything known to the emap.
Either this flag should be removed (and we are back to where we were yesterday) or the logic should propagate down to the unpacker implementation.